index.bs

```diff
@@ -1317,6 +1317,8 @@ A [=view=] has an associated <dfn for="view">projection matrix</dfn> which is a
 
 A [=view=] has an associated <dfn for="view">eye</dfn> which is an {{XREye}} describing which eye this view is expected to be shown to. If the view does not have an intrinsically associated eye (the display is monoscopic, for example) this value MUST be set to {{XREye/"none"}}.
 
+A [=view=] has an <dfn for="view">active</dfn> flag that may change through the lifecycle of an {{XRSession}}. [=Primary views=] SHOULD always have the [=view/active=] flag set to <code>true</code>.
```

Comment:
Any reason why we don't say the primary views MUST always be active?

Reply:
Because that's a new constraint, the thing defining a primary view is that without it the system cannot perform. It's not impossible for devices to have primary views that are variable, it's just weird.